app/models/post.rb

```diff
@@ -152,4 +149,10 @@ def comment_email_subject
   def nsfw
     self.author.profile.nsfw?
   end
+
+  def subscribers
+    subs = super
+    subs += resharers + participants if public?
```

Comment:
Do we own the object we get from the parent at this point? `Array.concat` could save a few array allocations here.